### ISSUE TITLE
Data values gets changed to 0-1 after reshaping the 1D to 2D from tab2img tool

### ISSUE THREAD

**Issue by EhtashamAhmed** [nicomignoni/tab2img#3]
Hi, I want to confirm that does this tool changes the data values as well? Or does it simply reshapes the data of 1D to 2D?

I am passing the data of 1x198 shape and it is returning me the data of 1x15x15 shape which is perfect square of the closest 2 dimension. But the data values are also getting changed between the range of 0-1

**Comment by nicomignoni** [comment]
No, data should not have their value changed, only reordered. If you attach the file containing your data, I can try to replicate your problem.

**Comment by EhtashamAhmed** [comment]
This is my first sample of input data of shape 197,
![image](https://user-images.githubusercontent.com/43585768/142061287-da6170de-7343-4bca-ba09-e37947ce1931.png)
And this is the reshaped data which i get
![image](https://user-images.githubusercontent.com/43585768/142061342-7cd9597d-0cae-4bc7-9e43-67c20a37e9cf.png)

**Comment by EhtashamAhmed** [comment]
This looks like very strange that when I run the code with the same data but with less features of 144 for the output of 12x12, my data does not change and the tab2img only reshapes the data

**Input data of 144 features**
![image](https://user-images.githubusercontent.com/43585768/142064997-46563ed9-9add-414f-ac68-04b2c8ce5773.png)

**Output contains the data reshaped to 12x12 and the original data values stayed the same as min value and max value**
![image](https://user-images.githubusercontent.com/43585768/142065162-5872facd-2b04-4ab6-9d65-c3035edf8ce7.png)


But if I increase the data features for the same dataset to 169 so that i get the output of 13x13, my data values gets changed

**Input data of 169 features**
![image](https://user-images.githubusercontent.com/43585768/142065353-59001757-fe04-45ec-a99b-92be1362e670.png)


**Output data is of shape 13x13 but the data values are now only 0 or 1**
![image](https://user-images.githubusercontent.com/43585768/142065439-2a8d2797-e386-48b0-8f66-683142a678de.png)
